swap/swap.go

```diff
@@ -40,17 +40,18 @@ import (
 )
 
 const (
```

Comment:
Could you clarify for me why these constants are in swap.go and not in defaults.go? What is the purpose of defaults.go and how do we decide when to put constants there versus in a different file?

Reply:
Good catch. It should all be there in `defaults.go`